src/introduction/prerequisites.md

```diff
@@ -52,7 +52,7 @@ environment:
 :name: installing-cwltool-with-pip-and-venv
 :caption: Installing `cwltool` with `pip` and `venv`.
 
-$ python -m venv venv
+$ python3 -m venv venv
```

Comment:
Either modify it here (and ensure we don't have any other `python` laying around) or we could add a note telling users in some cases their operating system may not include a `python` command, in which case they may need to use another Python 3 available in their environment, such as `python3`, `python3.7`, or `python3.10`. In my case I have `python3` and `python3.10` on my `$PATH`.

Leaving it up to other reviewers to choose :+1: 

Thanks for the PR!

Reply:
@mr-c said to modify it here and in contributing.md, and that since python3 works where python is supposed to work, python3 should be used instead.

 About other `python` lying around, I will check through the user guide and make sure there aren't more.